### ISSUE TITLE
 I got poor results on my own screenshots. 

### ISSUE THREAD

**Issue by NHZlX** [ritheshkumar95/im2latex-tensorflow#18]
I don't know why I got poor results on my own screenshots, help welcome

**Comment by qianwangn** [comment]
same problem:  also got poor results on this test set

**Comment by qianwangn** [comment]
when you read image on your own screenshots,try to -54 for each channel, that may help
or you can train your own data from scratch 
thats two solution may solve your problem

**Comment by Zack-Tsang** [comment]
Hi，I met a strange wrong, do you know how to fix with it?
![image](https://user-images.githubusercontent.com/39196046/62816523-9489fd00-bb5b-11e9-9d57-72c87fe532ac.png)

**Comment by pageedward** [comment]
@qianwangn 
![4cbdf71aaf](https://user-images.githubusercontent.com/26244036/123361771-36753000-d5a2-11eb-9aa9-gt:   E = v \sqrt { Q _ { M } ^ { 2 } + Q _ { E } ^ { 2 } } \, \, .
ret:  \circ \circ \circ \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \a
lpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \r
ho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alp
ha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho
 \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho \alpha \rho
i use predict.py but get a very bad result. do u solve it?
